### PR TITLE
add job memory limit faq

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds a job memory limit faq for support and users who encounter this error. refer to slack: https://dbt-labs.slack.com/archives/C02NCQ9483C/p1720498592885069

### CODE REVIEW

File: website/docs/faqs/Troubleshooting/job-memory-limits.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: "I'm receiving a 'This run exceeded your account's run memory limits' error in my failed job"
+description: "Use incremental models or optimize queries for job failures due to exceeded memory limits."
+sidebar_label: 'Job failures due to exceeded memory limits'
+---
+
+If you're receiving a `This run exceeded your account's run memory limits` error in your failed job, it means that the job exceeded the memory limits set for your account. Memory limits are on a per run basis and are typically influenced by the amount of result data that dbt has to ingest and process, which typically is quite small but can be bloated unexpectedly by project design choices.
+
+## Reasons
```
Comment (mirnawong1) [inline website/docs/faqs/Troubleshooting/job-memory-limits.md]: maybe this is better?
```suggestion
## Common reasons
```

File: website/docs/faqs/Troubleshooting/job-memory-limits.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: "I'm receiving a 'This run exceeded your account's run memory limits' error in my failed job"
+description: "Use incremental models or optimize queries for job failures due to exceeded memory limits."
+sidebar_label: 'Job failures due to exceeded memory limits'
+---
+
+If you're receiving a `This run exceeded your account's run memory limits` error in your failed job, it means that the job exceeded the memory limits set for your account. Memory limits are on a per run basis and are typically influenced by the amount of result data that dbt has to ingest and process, which typically is quite small but can be bloated unexpectedly by project design choices.
+
```
Comment (nataliefiann) [inline website/docs/faqs/Troubleshooting/job-memory-limits.md:8]: Hiya @mirnawong1 

This is a non-blocker suggestion, but would it be worth letting customers know all dbt Cloud accounts have a pod memory of 600Mib set?

Kind Regards

File: website/docs/faqs/Troubleshooting/job-memory-limits.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- dbt run/build:  Macros that capture large result sets from run query may not all be necessary and may be memory inefficient.
+- dbt docs generate: Source or model schemas with large numbers of tables (even if those tables aren't all used by dbt) cause the ingest of very large results for catalog queries.
+
+## Resolution
+
+To resolve this, you can try the following:
+
+1. **Use incremental models**: Try using [incremental models](/docs/build/incremental-models-overview) to reduce the amount of data being processed in each run. Incremental models only process new or updated data, which can help reduce the memory usage of your jobs.
+2. **Refactor your data model**: Review your data models to see if there are any opportunities to optimize or refactor them. For example, you can try to reduce the number of columns being selected, use `where` clauses to filter data early in the query or use `limit` clauses to reduce the amount of data being processed.
+
```
Comment (nataliefiann) [inline website/docs/faqs/Troubleshooting/job-memory-limits.md:22]: Hiya @mirnawong1 

Another non-blocker suggestion, I have some some links to some docs on run memory which you could refer in this one:

https://docs.getdbt.com/docs/deploy/job-scheduler#job-memory
 
https://docs.getdbt.com/blog/how-we-shaved-90-minutes-off-model

Kind Regards
Natalie

File: website/docs/faqs/Troubleshooting/job-memory-limits.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+title: "I'm receiving a 'This run exceeded your account's run memory limits' error in my failed job"
+description: "Use incremental models or optimize queries for job failures due to exceeded memory limits."
+sidebar_label: 'Job failures due to exceeded memory limits'
+---
+
+If you're receiving a `This run exceeded your account's run memory limits` error in your failed job, it means that the job exceeded the memory limits set for your account. Memory limits are on a per run basis and are typically influenced by the amount of result data that dbt has to ingest and process, which typically is quite small but can be bloated unexpectedly by project design choices.
```
Comment (mirnawong1) [inline website/docs/faqs/Troubleshooting/job-memory-limits.md:7]: add link to this faq directly in the error message: https://github.com/dbt-labs/dbt-cloud/blob/378f5a06d18998c367d1c4a3473cd607f98f2b08/sinter/common/error_reasons.py#L3

File: website/docs/faqs/Troubleshooting/job-memory-limits.md (hunk 1 above)
Comment (nataliefiann) [reply website/docs/faqs/Troubleshooting/job-memory-limits.md]: Hiya @mirnawong1 

I quite like **Common reasons** as a title for the section.
**Common reasons** indicate these are the reasons we've come across for the job error and the steps in the doc might help. Whereas **Reasons** indicates THIS IS / THESE ARE the reasons which kinda goes against the section on reaching out to support if the troubleshooting steps don't work.
Just my two cents.

Kind Regards
Natalie

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thanks for re-flagging this and the use of the blog link might be good for users. Thanks for adding this. I've left a note regarding the title section Common reasons vs Reasons but it's not a blocker. I'll approve this for you now.

Kind Regards

**Pull request merged**